srcpkgs/genie/template

```diff
@@ -0,0 +1,48 @@
+# Template file for 'genie'
+pkgname=genie
+version=1141
+revision=1
+_githash=44918162588e56512ddac6264b08ca6ba4e67468
+hostmakedepends="git"
+short_desc="Project generator tool using Lua"
+maintainer="a dinosaur <nick@a-dinosaur.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/bkaradzic/GENie"
+
+do_fetch() {
+	git clone "${homepage}.git" "${pkgname}-${version}"
+	cd "${pkgname}-${version}"
+	git reset --hard "${_githash}"
+}
```

Comment:
please `distfiles="https://github.com/bkaradzic/GENie/archive/${_githash}.tar.gz"` instead of `git clone`

Reply:
Done.